LiteCore/Database/DatabaseImpl.cc

```diff
@@ -147,11 +147,12 @@ namespace litecore {
         // Set up DataFile options:
         DataFile::Options options { };
         options.keyStores.sequences = true;
-        options.create = (_config.flags & kC4DB_Create) != 0;
-        options.writeable = (_config.flags & kC4DB_ReadOnly) == 0;
-        options.upgradeable = (_config.flags & kC4DB_NoUpgrade) == 0;
-        options.useDocumentKeys = true;
-        options.diskSyncFull = (_config.flags & kC4DB_DiskSyncFull) != 0;
+        options.create              = (_config.flags & kC4DB_Create) != 0;
+        options.writeable           = (_config.flags & kC4DB_ReadOnly) == 0;
+        options.upgradeable         = (_config.flags & kC4DB_NoUpgrade) == 0;
+        options.diskSyncFull        = (_config.flags & kC4DB_DiskSyncFull) != 0;
+        options.noHousekeeping      = (_config.flags & kC4DB_NoHousekeeping) != 0;
+        options.useDocumentKeys     = true;
```

Comment:
Why switch the order of useDocumentKeys and diskSyncFull?

Reply:
I did not switch the order. I tried to use whatever the order in original 3.1 source.